substrate/frame/statement/Cargo.toml

```diff
@@ -49,4 +49,4 @@ try-runtime = [
 	"frame-system/try-runtime",
 	"pallet-balances/try-runtime",
 	"sp-runtime/try-runtime",
-]
+]
```

Comment:
Please revert all these changes that are not relevant for this pr.

Reply:
Hello! @bkchr 
I made use of one PR for the two pallet changes. Can you review it there,
 please? https://github.com/paritytech/polkadot-sdk/pull/7106